### PR TITLE
🔒 DT-2565: Fixing XSS issue in external user search

### PR DESCRIPTION
**Pull request by brightonsbox**

### CODE REVIEW

File: integration-tests/support/externaluser.helpers.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const UserSearchResultsPage = require('../pages/userSearchResultsPage')
 
 export const replicateUser = (times) =>
   [...Array(times).keys()].map((i) => ({
+    userId: i,
```
Comment (brightonsbox) [inline integration-tests/support/externaluser.helpers.js:7]: This was missing so was showing up as undefined in the URL.

File: integration-tests/integration/externaluser.manage.spec.js [hunk 2]
```diff
@@ -358,6 +358,7 @@ context('External user manage functionality', () => {
   it('Should change a user email address when username same as email address', () => {
     const results = searchForUser(undefined, [
       {
+        userId: 1,
```
Comment (brightonsbox) [inline integration-tests/integration/externaluser.manage.spec.js:361]: This was missing so was showing up as undefined in the URL.

File: integration-tests/integration/externaluser.search.spec.js [hunk 3]
```diff
@@ -269,6 +286,7 @@ context('External user search functionality', () => {
       const validateCsv = (list) => {
         expect(list, 'number of records').to.have.length(22)
         expect(list[0], 'header row').to.deep.equal([
+          'userId',
```
Comment (brightonsbox) [inline integration-tests/integration/externaluser.search.spec.js:289]: I checked that these were indeed included in the CSV in dev, just missing in the tests but shows up correctly now.

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**